api/src/test/java/uk/gov/dhsc/htbhf/claimant/repository/ClaimantRepositoryTest.java

```diff
@@ -68,76 +69,98 @@ void shouldHandleSqlInjectionAttempts() {
 
         //Then
         assertThat(claimantExists).isFalse();
     }
 
-    @Test
-    void shouldReturnClaimantDoesNotExistWhenNinoExistsAndStatusIsNotEligible() {
+    @ParameterizedTest(name = "Should return that a live claim does exist when claimant exists and claim status is {0}")
+    @ValueSource(strings = {
+            "NEW",
+            "ACTIVE",
+            "PENDING",
+            "PENDING_EXPIRY"
+    })
+    void shouldReturnLiveClaimDoesExistWhenNinoExists(ClaimStatus claimStatus) {
         //Given
-        Claimant claimant = aValidClaimantBuilder().eligibilityStatus(INELIGIBLE).build();
+        Claimant claimant = aValidClaimantBuilder().claimStatus(claimStatus).build();
         claimantRepository.save(claimant);
 
         //When
-        Boolean claimantExists = claimantRepository.eligibleClaimExistsForNino(claimant.getNino());
+        Boolean claimantExists = claimantRepository.liveClaimExistsForNino(claimant.getNino());
 
         //Then
-        assertThat(claimantExists).isFalse();
+        assertThat(claimantExists).isTrue();
     }
 
-    @Test
-    void shouldReturnClaimantExistsWhenNinoExistsAndStatusIsEligible() {
+    @ParameterizedTest(name = "Should return that a live claim does not exist when claimant exists and claim status is {0}")
+    @ValueSource(strings = {
+            "REJECTED",
+            "ERROR",
+            "EXPIRED"
+    })
+    void shouldReturnLiveClaimDoesNotExistWhenNinoExists(ClaimStatus claimStatus) {
         //Given
-        Claimant claimant = aValidClaimantBuilder().eligibilityStatus(ELIGIBLE).build();
+        Claimant claimant = aValidClaimantBuilder().claimStatus(claimStatus).build();
         claimantRepository.save(claimant);
 
         //When
-        Boolean claimantExists = claimantRepository.eligibleClaimExistsForNino(claimant.getNino());
+        Boolean claimantExists = claimantRepository.liveClaimExistsForNino(claimant.getNino());
 
         //Then
-        assertThat(claimantExists).isTrue();
+        assertThat(claimantExists).isFalse();
     }
 
     @Test
-    void shouldReturnClaimantDoesNotExistWhenDwpHouseholdIdentifierDoesNotExist() {
+    void shouldReturnLiveClaimDoesNotExistWhenDwpHouseholdIdentifierDoesNotExist() {
         //Given
         Claimant claimant = aValidClaimantBuilder().build();
 
         //When
-        Boolean claimantExists = claimantRepository.eligibleClaimExistsForDwpHousehold(claimant.getDwpHouseholdIdentifier());
+        Boolean claimantExists = claimantRepository.liveClaimExistsForDwpHousehold(claimant.getDwpHouseholdIdentifier());
 
         //Then
         assertThat(claimantExists).isFalse();
     }
 
-    @Test
-    void shouldReturnClaimantDoesNotExistWhenDwpHouseholdIdentifierExistsAndStatusIsNotEligible() {
+    @ParameterizedTest(name = "Should return a live claim does not exists when dwp household exists and the claim status is {0}")
```

Comment:
Grammar check, you probably meant to say `Should return that a live claim does not exist...`

Reply:
Done.